libs/blocks/media/media.js

```diff
@@ -49,6 +55,44 @@ export default function init(el) {
     if (image) image.classList.add('image');
     const img = image.querySelector(':scope img');
     if (header && img?.alt === '') img.alt = header.textContent;
+
+    // lists
+    if (row.querySelector('ul')) {
+      el.querySelector('ul').classList.add('default-list');
+    }
+
+    // subcopy
+    const actionArea = row.querySelector('p.action-area');
+    if (actionArea?.nextElementSibling?.tagName === 'P') {
+      updateInnerHtml(actionArea.nextElementSibling, 'strong', '');
+      actionArea.nextElementSibling.className = 'subcopy';
+    }
+
+    // subcopy with links
+    if (actionArea?.nextElementSibling?.tagName === 'H3') {
+      actionArea.nextElementSibling.classList.add('subcopy-heading');
```

Comment:
Maybe we can change the typography class `body-xl` to `heading-xs` on this element and try to align w/ typography.css classes where we can.

Reply:
Generally I try to keep the classes listed in an element honest. This starts to become hard to maintain when there are some overrides happening and conflicting name spaces. 
<img width="1517" alt="Screen Shot 2023-07-25 at 2 10 20 PM" src="https://github.com/adobecom/milo/assets/2086146/a4910d88-724e-42ac-8e8e-c38fa4f9563e">

Reply:
Resolved https://github.com/adobecom/milo/pull/971/commits/40626583cf40796a247d6a1b33e314ade1fac93c @ryanmparrish